### ISSUE TITLE
Bugs in the latest version of Sound manager

### ISSUE THREAD

**Issue by MichalSuchy** [yplassiard/nvda-soundmanager#4]
Hello. I installed the latest version of Sound manager 2019.5.6.1 and I found some bugs. It still doesn't work muting master volume. When I am not in dialog for Toggle volume control adjustment and I press ctrl+m, Nvda says explorer is not playing any sound. Nvda says same notification in other applications, which I know, that they play sounds too. It is for example mail program Outlook express. And I found another bug. When I am in dialog for Toggle volume control adjustment, there aren't all applications in circle, which are in Windows mixer. There are 3 or 4 applications, when I pressing left or right arrow.

**Comment by yplassiard** [comment]
### Need Clarification
Hi Michal,
This issue needs some clarifications, as several bugs are described inoto a single entry. As far as I understand, it seems that some applications do not appear into the "app circle" of this add-on. This is due to the fact that only applications that are currently plaing sounds are shown here, not all running apps. If for example you launch Google Chrome and play a Youtube video, it will show up in the app circle. But, if for some reasons your Chrome do not play any sound  for a while, the app will disappear from the circle until a new sound is being played for this app.
I'll update the documentation to state this behavior more clearly.
